docs/tutorials/basic_commands.md

```diff
@@ -28,32 +24,31 @@ functionalites of the tool. They are:
 * **disable**
 
 In this tutorial, we will go through all those commands to show how to properly use them.
-To achieve that, we will use a Xenial LXD container.
+To achieve that, we will use a Xenial Multipass virtual machine.
```

Comment:
What's the difference between a Xenial Multipass VM and a regular one? I found the change in terminology throughout to be a bit confusing - sometimes you refer to the Multipass VM, and sometimes to the Xenial one. Are they not the same thing?

Reply:
There is none. I am changing the text to only use `Xenial Multipass VM`. But let me know is you think this is too verbose

Reply:
Probably what I would recommend in that case is using "Xenial Multipass VM" for the first time you refer to Multipass, and just mentioning that thereafter you'll be calling it whatever you want to call it (and only using that for the rest of the page). As long as it's defined right at the start, it shouldn't cause any problems.